policy-packs/aws-ts-finops/rds.ts

```diff
@@ -0,0 +1,115 @@
+import * as aws from "@pulumi/aws";
+import {
+  ReportViolation,
+  ResourceValidationArgs,
+  ResourceValidationPolicy,
+  StackValidationPolicy,
+  EnforcementLevel,
+  validateResourceOfType,
+} from "@pulumi/policy";
+import * as utils from "./utils";
+
+//Burstable
```

Comment:
What does this comment mean?

Reply:
I was going to do burstable only, but then I just did "Instance types" to make it broader

Reply:
I removed the comment.